pkg/scheduler/extender_test.go

```diff
@@ -488,7 +496,9 @@ func TestConvertToVictims(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		tt:=tt
```

Comment:
```suggestion
		tt :=  tt
```

there has to be an space after

Reply:
Oops, I forgot to fix the format